### ISSUE TITLE
Display goes white intermittently when embedded is used

### ISSUE THREAD

**Issue by sheadarlison** [Rise-Vision/rise-embedded-template#17]
Details: https://rise-vision-inc.slack.com/archives/CCELMSZFZ/p1587127416058800

**Comment by olegrise** [comment]
Fixed. The problem was that `rise-playlist` component was not receiving the `rise-presentation-play` event from `common-template`. More details in the [PR](https://github.com/Rise-Vision/common-template/pull/117)

@alex-deaconu @ezequielc just FYI, I've noticed that when I do the following: 
- click on save schedule in the Apps
- wait for Viewer to reload content
- repeat
and if I do that 6-10 times then something happens to the Viewer and it starts sending `rise-presentation-stop` right after `rise-presentation-play` as if duration was set to 0. As a result components that rely on those events (for example rise-playlist) do not show content. I saw this on Chrome OS player. This is just a note in case we need to investigate further in the future.
